keras/backend/mxnet_backend.py

```diff
@@ -2741,11 +2814,20 @@ def categorical_crossentropy(target, output, from_logits=False):
     # Returns
         Output tensor.
     """
-    assert not from_logits
     axis = ndim(output) - 1
     mx_output = output.symbol
+    # scale predictions so that the class probas of each sample sum to 1
+    if from_logits:
```

Comment:
Can you please enable categorical cross entropy tests.

Reply:
Comments?

Reply:
I enabled the categorical crossentropy testcases. Thanks!